src/gax/src/options/mod.rs

```diff
@@ -192,20 +192,25 @@ pub trait RequestOptionsBuilder {
     fn with_polling_backoff_policy<V: Into<PollingBackoffPolicyArg>>(self, v: V) -> Self;
 }
 
-/// Simplify implementation of the [RequestOptionsBuilder] trait in generated
-/// code.
-///
-/// This is an implementation detail, most applications have little need to
-/// worry about or use this trait.
-pub trait RequestBuilder {
-    fn request_options(&mut self) -> &mut RequestOptions;
+/// This module contains implementation details. It is not part of the public
+/// API. Types inside may be changed or removed without warnings. Applications
+///  should not use any types contained within.
+#[doc(hidden)]
+pub mod internal {
```

Comment:
Does the module need to be public?  Could you say `mod internal` ?

Reply:
It needs to be public. All of the builders use it.

https://github.com/googleapis/google-cloud-rust/blob/d35df5e8ddb145c0ab0efe78c681ccbf11c11df2/generator/testdata/rust/openapi/golden/src/builders.rs#L106